frontend/src/tests/import-export.test.js

```diff
@@ -190,4 +196,245 @@ describe("Import/export library functionality", () => {
         const ddahJson = ddahData.map(prepareMinimal.ddah);
         expect(ddahJson).toMatchSnapshot();
     });
+
+    it("Import Instructors from JSON/CSV/XLSX", () => {
+        // import correct instructors from XLSX
+        const normalizedSpreadsheetInstructors = normalizeImport(
+            {
+                fileType: "spreadsheet",
+                data: parseSpreadsheet("instructors_correct.xlsx"),
+            },
+            instructorSchema
+        );
+        expect(normalizedSpreadsheetInstructors).toMatchSnapshot();
+        // import correct instructors from JSON
+        const normalizedJsonInstructors = normalizeImport(
+            {
+                fileType: "json",
+                data: objectJSON["instructors"],
```

Comment:
Why aren't you doing `.instructors` to access this? Also, what does `objectJSON` mean? I don't understand what data `objectJSON` is supposed to contain.